uint8 max_hue
uint8 max_sat
uint8 max_value
uint8 min_hue
uint8 min_sat
uint8 min_value
